Userland/Applets/Audio/main.cpp

```diff
@@ -201,7 +202,7 @@ class AudioWidget final : public GUI::Widget {
     Gfx::Bitmap& choose_bitmap_from_volume()
     {
         if (m_audio_muted)
-            return *m_volume_level_bitmaps.last().bitmap;
+            return *m_volume_level_bitmaps.back().bitmap;
```

Comment:
Not related to your changes, but it's weird to me that Array uses `front()/back()` instead of `first()/last()` like everything else. :yakthonk: I might have to fix it since it's going to bother me now!